src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,61 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplate(versionStr: string, studio: string, from: string, to: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplate.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
```

Comment:
Wouldn't this give the same result before and after the migration since you don't change the values of gfxTemplates in this migration?

Reply:
This is no longer relevant as of [73a488c](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/73a488c30f009df70d38e047371d4b47fdf749ec), since the two migrations have been merged.